docs/cli/start/workflows.md

```diff
@@ -39,8 +39,8 @@ During the init process, the root stack is created with three resources:
 The provider logs the information of the root stack and the resources into the project metadata file (amplify/backend/amplify-meta.json).
 The root stack's template can be found in `amplify/backend/awscloudformation`.
 
-### amplify <category> add
-Once init is complete, run the command `amplify <category> add` to add resources of a category to the cloud. This will place a CloudFormation template for the resources of this category in the category's subdirectory `amplify/backend/<category>` and insert its reference into the above-mentioned root stack as the nested child stack. When working in teams, it is good practice to run an `amplify pull` before modifying the backend categories.
+### amplify \<category\> add
```

Comment:
If we do maintain a valid tags set, do we still need to escape or can we just infer that this is text at build time.

Reply:
We need to escape it, otherwise it'll produce an invalid tag error at build time.

Reply:
ok